rancilio-pid/rancilio-pid.ino

```diff
@@ -481,15 +484,23 @@ void refreshTemp() {
   }
   if (TempSensor == 2)
   {
-    if (currentMillistemp - previousMillistemp >= intervaltempmestsic)
+    unsigned long millis_elapsed = currentMillistemp - previousMillistemp ;
```

Comment:
imho unnötiges "feature", sogut wie niemand der user hat die Maschine am PC.
Das ist zum Entwickeln eventuell hilfreich, sollte aber nicht in den finalen Code.

Reply:
Ich habe bereits ein paar mal diese Fehler im Test-Betrieb gefunden und daher ist das Logging hier eingebaut. In der aktuellen Version mit MQTT-Integration geht an dieser Stelle auch ein Event über MQTT raus, sodass man prüfen kann was los ist. 

Ich würde begrüssen wenn man Code den man dann zum DEBUGGEN braucht, hier drinnen lässt, damit diejenigen die neue Features entwickelt sofort merken wenn etwas nicht funktioniert wie es sollte. Das führt letztendlich zu einer höheren Stabilität unseres Codes. (Der Nutzen für reine Endanwender ist gering, da hast du Recht).

Reply:
Done

Reply:
muss das nun in 1.9.7 übernommen werden?